widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -573,15 +571,21 @@ div#promptPanel div.prompt-panel {
 }
 
 div#toppanel {
-  margin-top: 20px;
+  margin-top: unset;
```

Comment:
Report Prompts are out-of-scope, AFAIK.
Also, these changes would affect prompts even when used outside the Parameter UI.

Reply:
Reverted the change.